shub/settings/config.py

```diff
@@ -159,3 +179,89 @@
     #    'globus',
     #    'saml_auth'
 ]
+
+# Any plugins enabled from the environment?
+PLUGINS_ENABLED += get_sregistry_envar_list("SREGISTRY_PLUGINS_ENABLED")
+
+## Environment
+
+# Custom additions are handled before this
+# Limit defaults that user can set
+BOOLEAN_DEFAULTS = [
+    "ENABLE_GOOGLE_AUTH",
```

Comment:
Perhaps alpha-sorting these lists?

Reply:
ah missed this! The current ordering (in settings.py) is my best effort to have them grouped logically by function (within a particular variable type, and over alpha sorting). Let me know if you think this should be done differently.

Reply:
Your current version where they are logically grouped (with comments!) is far better and we should keep what you have now.